db/re/item_db_etc.yml

```
@@ -89439,7 +89470,12 @@ Body:
     AegisName: aegis_1001615
     Name: Footprint Effect (Garment)
     Type: Etc
-    Weight: 100
+    Weight: 10
```

`#   Weight                  Item weight. Each 10 is 1 weight. (Default: 0)`
```suggestion
    Weight: 100
```

---

item info from client say 10
"무게 : ^777777 10 ^000000"